tests/FsCheck.Test/Gen.fs

```diff
@@ -119,6 +119,13 @@ module Gen =
         Gen.listOfLength length (Gen.constant v)
         |> sample1
         |> ((=) (List.init length (fun _ -> v)))
+
+    [<Property>]
+    let Shuffle (xs:Set<int>) =
+        not xs.IsEmpty ==> lazy
+        Gen.shuffle xs
+        |> sample 10
+        |> List.exists ((<>) (Set.toSeq xs))
```

Comment:
@ploeh True, but then by applying the Devil's Advocate technique, the implementation may as well be this?

```
let shuffle xs = xs
```

:confused: